### DISCUSSION TITLE
Symfony Does Not Work With Apache Web Server | Symfony Version - 6.4.3 | PHP Version - 8.1.28

### DISCUSSION THREAD

**Discussion by BharathKumarZiffity** [symfony/symfony#54867] [Q&A] [answered]
Hi there i have recently installed a symfony trying to do a local setup , but i'm not getting into any other pages using a URL path , for example let's say this is my domain -> local-symfony.com , if i want to go into this page -> local-symfony.com/test , then this leads to 404 error page using apache web server , but not in nginx.

Please help me in solving this issue , thanks !!

**Comment by Trismegiste** [comment]
You seem to have a problem with your rewrite rules

You can try the apache pack :
https://symfony.com/components/Apache%20Pack

**Reply by BharathKumarZiffity** [reply]
Hi , thank you for your suggestions , however after installing this pack using composer , the .htaccess file is not showing up in public directory , is there any specific commands i should run for generating it and symfony version and php version i have mentioned already.

**Comment by Trismegiste** [comment, marked as answer]
After the installation of this package, the recipe should copy this file https://github.com/symfony/recipes-contrib/blob/main/symfony/apache-pack/1.0/public/.htaccess into `%project_dir%/public`

Of course, it's always better to copy those rewriting rules into the site confg stored in `/etc/apache/available-sites/` instead of relying on `.htaccess`
